app/uk/gov/hmrc/customs/rosmfrontend/controllers/HasExistingEoriController.scala

```diff
@@ -41,6 +45,20 @@ class HasExistingEoriController @Inject()(
         Future.successful(Ok(hasExistingEoriView(service, existingEori.id)))
   }
 
+  def enrol(service: Service.Value): Action[AnyContent] =
+    ggAuthorisedUserWithEnrolmentsAction { implicit request => user: LoggedInUserWithEnrolments =>
+      enrolmentService.enrolWithExistingCDSEnrolment(user, service.toString.toLowerCase).map {
```

Comment:
i thought about changing the service to accept a 'Service.Value'.  But left it for now.

Reply:
Yeah, always we can implement this Service enumaration with sealed trait with value which we will be passing to the enrolment methods

Reply:
I might look at this as a separate code refactor PR after this has gone it